keras_tuner/distribute/oracle_chief_test.py

```diff
@@ -174,12 +174,5 @@ def _test_get_best_trials():
                 client.update_trial(trial_id, {"score": score})
                 client.end_trial(trial_id)
                 trial_scores[trial_id] = score
-            return
-            best_trials = client.get_best_trials(3)
-            best_scores = [t.score for t in best_trials]
-            assert best_scores == [9, 8, 7]
-            # Check that trial_ids are correctly mapped to scores.
-            for t in best_trials:
-                assert trial_scores[t.trial_id] == t.score
```

Comment:
Please add these lines back.
They were skipped by accident.

Reply:
I have added it back! The question is, will the _empty_ `return` prevent that `best_trials = client.get_best_trials(3)` and the rest will be reached?

Reply:
Yes, I mean the "return" should be removed.
My comment is actually pointing the lines other than the "return" : )

The PR is merged.
If you are interested, you can have another PR to remove the return so that the rest of the lines are reachable.